packages/accessibility/global.d.ts

```diff
@@ -1,8 +1,13 @@
-declare namespace GlobalMixins
+declare global
 {
-    // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    interface DisplayObject extends Partial<import('@pixi/accessibility').IAccessibleTarget>
+    namespace GlobalMixins
     {
+        // eslint-disable-next-line @typescript-eslint/no-empty-interface
+        interface DisplayObject extends Partial<import('@pixi/accessibility').IAccessibleTarget>
+        {
 
+        }
     }
 }
+
+export {};
```

Comment:
this turns a file into a "module", which means that to declare a global you need to use `declare global` statement. It just helps dts-bundle-generator differentiate between "local" and "global" declarations. changes in all global files are pretty much the same - wrapping with `declare global` and adding `export {}` (apart from few files)